### ISSUE TITLE
Objects order in composite

### ISSUE THREAD

**Issue by zaynyatyi** [UnderwaterApps/overlap2d#356]
In composite object when i'm adding image first and then polygon there is no way how to change order of that objects (put polygon below image or in another way)

**Comment by azakhary** [comment]
Wait, does changing z-indexes help? Pressing Ctrl+ Up or down keys. Is it polygon specific thing?

**Comment by zaynyatyi** [comment]
Yeah, that works. But a bit unclear. Maybe we should add it to context menu with hotkey hint?

**Comment by azakhary** [comment]
Agreed, would be nice to add it as a hotkey.

**Comment by skusku** [comment]
I would love to have a Textbox to put this into, close to the object's dimensions.

**Comment by azakhary** [comment]
You mean in basic Properties of every item?

**Comment by azakhary** [comment]
Usually it's bad idea to "set" a particular number, because that then prepositions all other items in z-index que, and this can mess things up. + No software I know does that.

**Comment by skusku** [comment]
Oh okay, Unity does have Z-Ordering that way, so I thought it's a good idea, but Unity is a 3D engine and uses the Z-Ordering value only to know on what level of the Z-axis the object should be drawn. So maybe I'm messing up things there.
